Dockerfile

```diff
@@ -19,20 +19,23 @@ RUN ldconfig /opt/OpenBLAS/lib/
 ##################### PREREQUISITES ########################
 
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update && apt-get install -y pkg-config m4 zlib1g libplplot-dev plplot-driver-cairo
-RUN apt-get remove plplot-driver-qt -y
-RUN opam install alcotest utop
+RUN apt-get update -y
+RUN apt-get install -y m4 wget unzip aspcud libshp-dev libplplot-dev gfortran pkg-config git
```

Comment:
Note that `aspcud` is no longer a dependency for opam, can be removed
I also don't see the need for `unzip`, `wget` and `git`

In any case we can get rid of both lines and replace them with `opam depext owl owl-plplot` right after `opam update -q`: this will install all the necessary dependencies.

If the image uses ubuntu-20.04, then it does not even need the custom openblas (not the docker image nor the custom compiled library and flags).